### PR TITLE
Core algorithm change + two small user interface changes for convenience

### PR DESCRIPTION
**Pull request by lemartinet**
- The overlap in the static part of DPP has been changed to m-1 in all cases based on feedback from Wes Viles.
- I changed the realtime output to write on the same line to avoid flooding the Matlab command window.
- I added a parameter to a function to provide more details when needed.

### CODE REVIEW

File: src/visualize/bendplot.m [hunk 1]
```diff
@@ -1,11 +1,16 @@
-function [] = bendplot(a, comm, circmat, xy)
+function [] = bendplot(a, comm, circmat, xy, labels)
 % Copyright Wes Viles 2013
 
     p = size(a,1);
     plot(xy(1,1),xy(1,2),'.','Color',[0,0,0],'MarkerSize',15);
     hold on
     for h = 1:p
         plot(xy(h,1),xy(h,2),'.','Color',[0,0,0],'MarkerSize',15);
+        if nargin == 5
```
Comment (nathanntg) [inline src/visualize/bendplot.m]: Using `nargin` for optional arguments is a bit error prone. For example, what if the order of arguments changes or a sixth argument is added? Any such change will require remembering to update this line.

Instead, can you use the `exist` function?

For example:

```
if exist('labels', 'var') && ~isempty(labels)
```

**Review by nathanntg — COMMENTED** [review]
Thanks for the PR, all the changes look good and glad we have a more consistent approach for the thresholding. I added one minor suggestion for the `bendplot` change.

**Comment by lemartinet** [comment]
ok, fixed it!

**Comment by nathanntg** [comment]
:+1: Looks great, thanks!

**Pull request merged**